src/Components/Home.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
+import './Home.css'
```

Comment:
We could use a `className` instead of writing CSS for this.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Thank you @EmmaDawsonDev I was still going through prime react on this

Reply:
Perhaps we could use a [`<NavLink>`](https://reactrouter.com/docs/en/v6/components/nav-link) instead of a `<Link>`?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient